core/scripts/testing/styles.css

```diff
@@ -48,3 +48,35 @@ html.ios.ios {
   --ion-font-family: -apple-system, BlinkMacSystemFont, "iosTestingFont", sans-serif;
   font-family: -apple-system, BlinkMacSystemFont, "iosTestingFont", sans-serif;
 }
+
```

Comment:
These classes are going to be useful for light (fallback) theme color testing as well.

I can move them to just the explicit tests if desired, but placing them in the shared test stylesheet cuts down on a lot of repeat boilerplate. These classes may also prove to be useful in testing other color contrast feature work.

Reply:
Never mind, these selectors directly conflict with the same selectors we use for color support in components. Adjusted the approach to just inject the styles for these tests: https://github.com/ionic-team/ionic-framework/pull/28609/commits/61dfd177718a8a110c70b0048be3bdb99d9c293f